data/job-board-rules-guidelines.md

```diff
@@ -0,0 +1,28 @@
+### 1. Job Submission and Approval Process:
+
+- After successful job submission, each job will undergo a verification process by the PyCon India 2023 team.
+- The approval process may take approximately 4-5 business days.
+- Approved jobs will be made visible on the job board once they pass the verification process.
+- Submissions that do not meet the guidelines may be rejected, and submitters will be notified of the reasons for rejection.
```

Comment:
Until when will the job board stay up, or do we plan to keep it there permanently?

Reply:
If we're taking 4-5 business days for a request, will we have a deadline after which it'll be too late to submit a job for it to be vetted properly and added by the time of the conference.?

Reply:
Do we have any restrictions on the industry/kinds of jobs we're allowing. Are we only allowing python related jobs?

Reply:
- Job board will stay even after the conference ends.
- The job posting once submitted can be reviewed in google sheet and marked as approved, as soon as that is done it will be published on webite.
- No restrictions on the job postings, may not be python specific only.